pkg/migrate/migrate.go

```diff
@@ -79,6 +106,37 @@ func Cli() {
 
 	output := log.New(os.Stdout, "", 0) // this has no time prefix etc
 
+	// escape hatch - for DEV/TEST ONLY
+	if !options.SkipPVAccessModeValidation {
+		srcPVs, err := kurlutils.PVSByStorageClass(context.TODO(), clientset, options.SourceSCName)
+		if err != nil {
+			fmt.Printf("failed to get volumes using storage class %s: %s", options.SourceSCName, err)
+			os.Exit(1)
+		}
+
+		pvMigrator := PVMigrator{
+			ctx:             context.TODO(),
+			log:             output,
+			k8scli:          clientset,
+			srcSc:           options.SourceSCName,
+			dstSc:           options.DestSCName,
+			deletePVTimeout: 5 * time.Minute,
+			podReadyTimeout: time.Duration(options.PodReadyTimeout) * time.Second,
+		}
+		unsupportedPVCs, err := pvMigrator.ValidateVolumeAccessModes(srcPVs)
+		if err != nil {
+			fmt.Printf("failed to validate volume access modes for destination storage class %s", options.DestSCName)
+			os.Exit(1)
+		}
+
+		if len(unsupportedPVCs) != 0 {
+			PrintPVAccessModeErrors(unsupportedPVCs)
+			fmt.Printf("existing volumes have access modes not supported by the destination storage class %s", options.DestSCName)
+			os.Exit(0)
```

Comment:
I'm wondering if there should even be an exit code here. If we return, then there's no way to "force" through a migration even though there are PVCs that are not supported.

Reply:
If we're exiting without having completed the migration, the exit code should not be 0.

Really though, this should be part of the 'Migrate' function, and not CLI.

(It should also probably be refactored into a function that is called from Migrate instead of having the logic on this level, IMO)

Reply:
@laverya 
>If we're exiting without having completed the migration, the exit code should not be 0.

We're failing the validation and not the _actual_ migration. This is why I exit here. Although you're right, probably should exit with a different return code.

>Really though, this should be part of the 'Migrate' function, and not CLI.

My original intention was to separate the validation and migration behaviors, logically. But given we don't want to depend on the kURL project's libraries I can now see this can be abstracted away.